src/webextension/settings/actions.js

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Strictly speaking, it's not necessary for the settings page to use Redux if you don't want. I've been hemming and hawing about where we should use Redux (aside from the management UI, where we really need it), so I'm open to suggestions.

In the longer term, if we do go with Redux here, we might want to pull out the Redux modal bits into a common place so that we're not duplicating as much code.

Reply:
Long-term, I'm not sure either.  For what this does _today_, it's kind of overkill.  For what I think we'll want in the future (automatic sign out options, filling behaviors, etc), it might still be needed.

If all of that necessitates Redux (which I suspect it will) then it felt like it would be more work to move things around later if we wanted more states than to just be ready for it now, hence the structure decisions I made.